manifests/init.pp

```diff
@@ -117,6 +117,12 @@
 # [*manage_user*]
 #   Whether to create/manage the user that should own consul's configuration files.
 #
+# [*manage_user_home_location*]
+#   Whether to explicitly set the location of teh consul user's home directory when this modules
+#   manages the creation of the user. If the consul user already exists and this is enabled, puppet
```

Comment:
Can you also note that this setting only takes effect if `manage_user` is true ?

Reply:
How does that look @solarkennedy ?